javascript/features/communication_commands/direct_communication_commands.js

```diff
@@ -4,6 +4,8 @@
 
 import { CommandBuilder } from 'components/command_manager/command_builder.js';
 import { ScopedCallbacks } from 'base/scoped_callbacks.js';
+import { MuteManager } from 'features/communication/mute_manager.js';
+import { relativeTime } from 'base/time.js';
```

Comment:
nit: please sort these alphabetically. We generally split up classes and functions in separate sections too:

```javascript
import { ClassName } from 'foo/class_name.js';
import { SecondClass } from 'foo/second_class.js';

import { functionName } from 'base/function.js';
```

Reply:
Fixed as of b10fd3ed14f79bc2e46618c53df2363d2a7ba280